src/components/NavBar/LanguageToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+import {Button, Text} from '@chakra-ui/react';
+import {FaGlobeAmericas} from 'react-icons/fa';
+
+/**
+ * Props for the Language Toggle
+ */
+interface LanguageToggleProps {
+  toggle: () => void;
+}
+
+/**
+ * Button to change the language between English and Spanish
+ * @param toggle - function that toggles the website language
+ * @returns a button that toggles the language when clicked
+ */
+const LanguageToggle: ({toggle}: LanguageToggleProps) => JSX.Element = ({
+  toggle,
+}: LanguageToggleProps) => {
+  const {t} = useTranslation('menu');
+  return (
+    <Button
+      variant="ghost"
+      _hover={{background: 'teal', fontWeight: 'bold'}}
```

Comment:
why is the `fontWeight` bold?

Reply:
this is what we want the `fontWeight` to be when we hover on the button (this matches the behavior of the links in the nav bar). You may have been referring to the fact that the `fontWeight` of the button was already semi-bold, which didn't match, so I made the default `fontWeight` regular and kept the `_hover` `fontWeight` bold

Reply:
ohh yes that was what I was referring to, sounds good!